lib/i18n_rails_helpers/i18n_model_helpers.rb

```diff
@@ -0,0 +1,7 @@
+module I18nModelHelpers
+  # translate enum fields value
+  def t_enum(enum_field)
+    I18n.t("activerecord.attributes.#{model_name.i18n_key}."\
+      "#{enum_field.to_s.pluralize}.#{public_send(enum_field)}")
+  end
+end
```

Comment:
It's better if the file path matches the class path, i.e. rename to `model_helpers.rb` and change the code to:

```ruby
module I18nRailsHelpers
  module ModelHelpers
    ...
```

(same for the controller helpers)